test/unittest/Database/resources/complex_dump.json

```diff
@@ -303,14 +303,6 @@
                 "samples_datas":{
                     "3":
                     [
```

Comment:
I think that an old dump file will be supported without issues, but we need a test to check it.